src/components/utils/GridLabelValue.js

```diff
@@ -10,17 +10,14 @@ import { Grid, Typography } from "@material-ui/core";
 
 // A grid where the secondary column, values, intentionally take up more
 // space than the first column, labels
-export default function GridLabelValue(props) {
-    const { label, children } = props;
-
+export default function GridLabelValue({ label, children, ...props }) {
     return (
         <>
             <Grid item sm={3} xs={6} md={3} lg={3}>
-                <Typography component="span" variant="subtitle2">
+                <Typography component="span" {...props}>
```

Comment:
Do we still want to leave `variant="subtitle2"` as a default setting? It can still be overridden by a `variant` prop passed by a parent component.
```suggestion
                <Typography component="span" variant="subtitle2" {...props}>
```

Reply:
Makes sense.